.gitignore

```diff
@@ -1,5 +1,7 @@
 .vscode/
-wurst/war3map.j
-objectEditingOutput/*
-compiled.j.txt
+/_build
```

Comment:
I wouldn't get rid of the built objects, they're super useful.

Please revert this.

Reply:
generated data does not belong into git

Reply:
@Frotty Fair argument, but unfortunately as the map still has some UI-based object data, this is the best way to view changes on that.

As discussed in IRC, I won't remove these build files until after I've extracted all those objects into compiletime fns